sdk/identity/Azure.Identity/src/DefaultAzureCredentialOptions.cs

```diff
@@ -10,6 +10,13 @@ namespace Azure.Identity
     /// </summary>
     public class DefaultAzureCredentialOptions : TokenCredentialOptions
     {
+        /// <summary>
+        /// The tenant id of the user to authenticate, in the case the <see cref="DefaultAzureCredential"/> authenticates through, the
+        /// <see cref="InteractiveBrowserCredential"/>. The default is null and will authenticate users to their default tenant.
+        /// The value can also be set by setting the environment variable AZURE_TENANT_ID.
+        /// </summary>
+        public string InteractiveBrowserTenantId { get; set; } = EnvironmentVariables.TenantId;
```

Comment:
Test that this is set from env var.

Reply:
I already updated the test which sets the `DefaultAzureCredentialOptions` through environment variables to also verify this property.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/646491ae3d836a6e986dd3101c21587b98eb30fe/sdk/identity/Azure.Identity/tests/DefaultAzureCredentialTests.cs#L117-L121